src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -20,15 +35,18 @@ function transformApplicants(applicants) {
   applicants.forEach(app => {
     const transformedApp = {
       fullName: app.applicantName ?? '',
-      sshOrTin: app.applicantSSN ?? '',
-      dateOfBirth: app.applicantDOB ?? '',
+      ssnOrTin: app.applicantSSN?.ssn ?? '',
+      dateOfBirth: fmtDate(app.applicantDOB) ?? '',
       phoneNumber: app.applicantPhone ?? '',
       email: app.applicantEmailAddress ?? '',
       vetRelationship: transformRelationship(
-        app.applicantRelationshipToSponsor?.relationshipToVeteran || 'NA',
+        app.applicantRelationshipToSponsor || 'NA',
```

Comment:
`transformRelationship` already drills down into `relationshipToVeteran`, so this was an unnecessary step